lib/msal-browser/src/interaction_handler/PopupHandler.ts

```diff
@@ -147,6 +141,25 @@ export class PopupHandler extends InteractionHandler {
         }
     }
 
+    static openSizedPopup(urlNavigate: string = "about:blank"): Window|null {
```

Comment:
For consistency's sake I would personally prefer we make this optional and have a line underneath here that checks if `StringUtils.isEmpty(urlNavigate)` and sets it to the default if it is

Reply:
I personally like using default values instead of `isEmpty`, more readable imo.

Reply:
+1 for this notation, it is a bit more readable.

Reply:
`StringUtils.isEmpty` check should still be added in case `openSizedPopup("")` is called.

Reply:
> `StringUtils.isEmpty` check should still be added in case `openSizedPopup("")` is called.

I'm not too worried about that happening, to be honest.

Reply:
Also, I believe calling `window.open` with `""` is functionality equivalent to `"about:blank"`

Reply:
This is fine. I generally agree with this being more readable if it's just one or a small number of params. I think my comment was more around keeping a consistent code pattern (I don't think I've seen this anywhere else in our code). If we want to start doing this more, I'm all for it 👍